Development

BenjaminGarza opened the pull request.

BenjaminGarza committed:
Added project info

BenjaminGarza committed:
Started site layout and positioning

BenjaminGarza committed:
Centered nav bar, started styling nav bar

BenjaminGarza committed:
Navbar gradient and styling

BenjaminGarza committed:
More css adjustments on the nav bar

BenjaminGarza committed:
Added headlines, more tweaks, some validation fixes

BenjaminGarza committed:
Positioning rework, added footer

BenjaminGarza committed:
Nav bar rework and styling

BenjaminGarza committed:
Small README updates

BenjaminGarza committed:
Removed end tag on line 24

Not needed

Oghenebrume50 commented:
Great start to this project. But some few changes need to be made

Oghenebrume50 commented on main.css:
```diff
@@ -0,0 +1,235 @@
+#main {
+  border: .01px transparent solid;
+  width: 1440px;
+  height:  805px;
```
This line is affecting your code, the footer jumps up. You may want to look into heights and width, most times you don't need to explicitly set heights of a large container. and your width is off too, it leaves a big space that is scrollable, too big to be ignored.

Oghenebrume50 commented on index.html:
```diff
@@ -0,0 +1,97 @@
+<body>
+   
+    <main id="main">
+       <article id="billboard">
```
This article tag shouldn't be wrapping the nav tag.

Oghenebrume50 commented on index.html:
```diff
@@ -0,0 +1,97 @@
+
+           </section>
+       </article>
+        <aside>
```
The content in this aside tag should be in the footer, and the images should be passed as background images, just like you did for the ones above. use a div and pass as background image

Oghenebrume50 reviewed (changes requested):
Fix those things and submit for re-review, nice attempt, you did really good.

BenjaminGarza committed:
Fixed footer, bugs, layout problems

Related content moved to footer Transparent border bug fixed. Removed Nav from article tag.

BenjaminGarza committed:
Sizing and layout tweaks

Adjusted sizing to get rid of scroll bar, various tweaks to make it display correctly after

Oghenebrume50 commented:
Hi, it is best to use positioning for inner elements and not outer container currently this is how your project displays
![11](https://user-images.githubusercontent.com/37092867/61896154-40044200-af0c-11e9-89a2-0f6d479e30ad.png)
**Tips**
Create an outer container to hold nav, main and footer Set a display: flex on it
Flex: direction column
avoid using margins on the nav, main, footer.
style the inner elements

Oghenebrume50 commented on main.css:
```diff
@@ -32,9 +31,10 @@ width: 100%;
     width: 62%;
-    height: 5%;
+    height: 30px;
     margin: 60px;
```
If you remove this margin you would see how it affects the page display. You have to be careful with margins

Oghenebrume50 commented on main.css:
```diff
@@ -0,0 +1,266 @@
+body{
+    
+    width:1440px;
```
You shouldn't set an explicit width like this on the body.

samgaco commented:
Hi, @BenjaminGarza I can't see any changes after the last review. Is everything ok?. I wait for your reply.

BenjaminGarza committed:
Made large layout and display changes

  Positioning is used on inner elements, outer elements are block level.
  Margins removed on block level elements.
  Removed explicit width on body

BenjaminGarza commented:
> Hi, @BenjaminGarza I can't see any changes after the last review. Is everything ok?. I wait for your reply.

Just made more changes today and putting in for another code review, thanks!

bolah2009 commented:
Currently under review

bolah2009 reviewed (approved):
## Nice Job [Ben](../../)  :clap: 

The [clone](https://raw.githack.com/BenjaminGarza/Backgrounds-and-Gradients/Development/index.html) looks good to me. :+1:

#### Consider checking these [errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FBenjaminGarza%2FBackgrounds-and-Gradients%2FDevelopment%2Findex.html) before merging.

#### Friendly reminder: You should use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in this project. However, changing it now would take too much time. Please make sure you use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in the next project. You will find instructions in the Pathwright lesson description.

Good luck with your next project.

Cheers
[Bola](https://bolabuari.com/)

BenjaminGarza committed:
Fixed strange behavior on resizing page, it is now a proper static site

Set minimum widths and fixed a problem with bottom promo pictures where they would overlap.

BenjaminGarza committed:
Wrapped input in ul in an li element (Validation error)
